### DISCUSSION TITLE
GET breaks when filter contains URI

### DISCUSSION THREAD

**Discussion by aabram** [pocketbase/pocketbase#4705] [Q&A] [answered]
I'm evaluating Pocketbase for my project and set up database with some test cases. Table 'posts' contains title, description, link, etc. From my dart code I want to check whether post with particular link already exists in the table, I use something like this:

```
  try {
    final result = await pb.collection('posts').getFirstListItem(
          'link = $link',
        );
  } catch (e) {
    print(e);
  }
```
  
This results in an error, --dev mode gives me:

```
ERROR GET /api/collections/posts/records?page=1&perPage=1&filter=link+=+https://foo.bar.com
└─ Something went wrong while processing your request.
   └─ Invalid identifier "https:"
```
   
As I understand the query breaks at the first /. Is there a workaround or am I doing it wrong? I'd rather not Uri.encodeComponent() all the links in posts table.

**Comment by ganigeorgiev** [comment, marked as answer]
The string value (`$link`) must be wrapped in single or double quotes.

**Reply by aabram** [reply]
Oh my, how did I miss that? Yes, with quotes it works as intended.
